.github/workflows/deploy-win.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build_dist:
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
this must stay windows-2019 to remain compatible with our production environment